consensus/src/subscriber.rs

```diff
@@ -13,6 +13,10 @@ use tokio::{
 use tracing::{debug, error};
 use types::{Certificate, CertificateDigest};
 
+#[cfg(any(test, feature = "benchmark"))]
```

Comment:
The `feature = "benchmark"` is not necessary and will probably break stuff

Reply:
good catch